src/main/resources/application.properties

```
@@ -53,6 +58,10 @@ quarkus.test.integration-test-profile=integrationtest
 %test,integrationtest.quarkusio.git-uri=file:tests-should-use-quarkus-io-sample-setup-annotation
 # disable scheduled indexing for dev/tests to not mess up anything with an unexpected reindexing:
 %dev,test,integrationtest.indexing.scheduled.cron=off
+# disable any error reporting to GitHub, simple logger will be used instead:
+%dev,test,integrationtest.indexing.error-reporting.enabled=false
+%dev,test,integrationtest.indexing.error-reporting.git-repository=not-a-user/not-a-repository
+%dev,test,integrationtest.indexing.error-reporting.git-issue-id=-1
```

It's a bit weird you have to put these placeholders. I think it would work with just this?

```suggestion
%dev,test,integrationtest.indexing.error-reporting.enabled=false
```

Or (probably clearer):

```suggestion
%dev,test,integrationtest.indexing.error-reporting.type=log
```

Though that should probably be the default, TBH.

And you'd use this for prod:

```
%prod.indexing.error-reporting.type=github-issue
%prod.indexing.error-reporting.github-issue.repository=quarkusio/search.quarkus.io
%prod.indexing.error-reporting.github-issue.id=89
```

I wonder how we'll handle staging, though... we'll probably need to introduce a separate config profile for staging, and a separate GitHub issue.